website/screens/common/sidenav/SidenavLogo.tsx

```diff
@@ -3,14 +3,12 @@ import Image from "@/common/Image";
 import halstackLogo from "@/common/images/halstack_logo.svg";
 import StatusTag from "@/common/StatusTag";
 import React from "react";
-
-const siteVersion = process.env.SITE_VERSION
-  ? typeof process.env.SITE_VERSION === "number"
-    ? `v${process.env.SITE_VERSION}.0.0`
-    : process.env.SITE_VERSION
-  : "dev";
+import { useRouter } from "next/router";
 
 const SidenavLogo = () => {
+  const router = useRouter();
+  const siteVersion = router.basePath.split("/")[2];
+
```

Comment:
This works with this URL: https://developer.dxc.com/halstack/next/, but does it also work with a URL like this: https://developer.dxc.com/halstack/next/components/application-layout/?

Also, as router is only used for getting the basePath I recommend to obtain the basePath like this, just to follow a good practice:
```
const { basePath } = useRouter();
const siteVersion = basePath.split("/")[2];
```

Reply:
> This works with this URL: https://developer.dxc.com/halstack/next/, but does it also work with a URL like this: https://developer.dxc.com/halstack/next/components/application-layout/?

`basePath` has the value from the config file `next.config.js`. In that path, we always have something like this:  `/halstack/SITE_VERSION`. Take a look at the [documentation](https://nextjs.org/docs/api-reference/next.config.js/basepath).

Reply:
Done requested change! ✌🏼